css/decanter.css

```diff
@@ -1020,21 +1020,21 @@ h6 {
   h4, .decanter-sans
   h5, .decanter-sans
   h6 {
-    font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif; }
+    font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Arial", sans-serif; }
   .decanter-serif h1, .decanter-serif
   h2, .decanter-serif
   h3, .decanter-serif
   h4, .decanter-serif
   h5, .decanter-serif
   h6 {
-    font-family: "Crimson Text", "Georgia", "Cambria", "Times New Roman", "Times", serif; }
+    font-family: "Crimson Text", "Georgia", "Times", "Times New Roman", serif; }
```

Comment:
Cambria is missing here.

Reply:
The fonts after `"Crimson Text"` here are generated by the `$font-stack-georgia` font stack variable. These are fallbacks. Bourbon has updated these.

Reply:
Thanks for confirming.